src/widgets/SensorStatus.js

```diff
@@ -75,45 +75,46 @@ export const SensorStatus = ({ isInHotBreach, isInColdBreach, isLowBattery, curr
 
   return !isInDanger ? (
     <Rectangle colour={WHITE}>
-      <BigText colour={DARKER_GREY}>{temperature(currentTemp).format()}</BigText>
+      <FlexRow alignItems="center">
+        <BigText colour={DARKER_GREY}>{temperature(currentTemp).format()}</BigText>
+        <TemperatureIcon />
+      </FlexRow>
     </Rectangle>
   ) : (
-    <TouchableOpacity>
-      <Rectangle colour={isInColdBreach ? COLD_BREACH_BLUE : DANGER_RED}>
-        <FlexRow flex={1}>
-          <FlexView alignItems="center" justifyContent="center">
-            <BigText>{temperature(currentTemp).format()}</BigText>
-          </FlexView>
-
-          <FlexView alignItems="center" justifyContent="center">
-            <Animated.View style={styles.icon} opacity={fadeAnim1}>
-              <HotBreachIcon />
-            </Animated.View>
-
-            <Animated.View style={styles.icon} opacity={fadeAnim2}>
-              <ColdBreachIcon />
-            </Animated.View>
-
-            <Animated.View style={styles.icon} opacity={fadeAnim3}>
-              <LowBatteryIcon />
-            </Animated.View>
-          </FlexView>
-        </FlexRow>
-      </Rectangle>
-    </TouchableOpacity>
+    <Rectangle colour={isInColdBreach ? COLD_BREACH_BLUE : DANGER_RED}>
+      <FlexRow flex={1}>
+        <FlexView flex={2} alignItems="center" justifyContent="center">
+          <BigText>{temperature(currentTemp).format()}</BigText>
+        </FlexView>
+
+        <FlexView alignItems="center" justifyContent="center">
+          <Animated.View style={styles.icon} opacity={fadeAnim1}>
+            <HotBreachIcon />
+          </Animated.View>
+
+          <Animated.View style={styles.icon} opacity={fadeAnim2}>
+            <ColdBreachIcon />
+          </Animated.View>
+
+          <Animated.View style={{ ...styles.icon, paddingRight: 5 }} opacity={fadeAnim3}>
+            <LowBatteryIcon size={20} />
+          </Animated.View>
+        </FlexView>
+      </FlexRow>
+    </Rectangle>
   );
 };
 
 const styles = {
-  icon: { position: 'absolute', left: 30 },
-  bigText: { fontSize: 32, fontFamily: APP_FONT_FAMILY },
+  icon: { position: 'absolute', alignSelf: 'flex-end' },
+  bigText: { fontSize: 24, fontFamily: APP_FONT_FAMILY },
 };
 
 SensorStatus.defaultProps = {
-  isInHotBreach: false,
-  isInColdBreach: false,
-  isLowBattery: false,
-  currentTemp: 0,
+  isInHotBreach: true,
+  isInColdBreach: true,
+  isLowBattery: true,
+  currentTemp: 22.22,
```

Comment:
Oops, certainly a mistake leftover from testing!

Reply:
Simultaneous hot and cold breach? haha

Reply:
d91b82d